lib/react_test_utils.dart

```diff
@@ -0,0 +1,227 @@
+      _Simulate.callMethod('touchStart', [element, eventData]);
+
+  static wheel(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('wheel', [element, eventData]);
+
+}
+
+
+/// Traverse all components in tree and accumulate all components where
+/// test(component) is true. This is not that useful on its own, but it's
+/// used as a primitive for other test utils
+///
+/// Included in Dart for completeness
+List findAllInRenderedTree(JsObject inst, JsFunction test) {
+  return _TestUtils.callMethod('findAllInRenderedTree', [inst, test]);
+}
+
+/// Like scryRenderedDOMComponentsWithClass() but expects there to be one
+/// result, and returns that one result, or throws exception if there is
+/// any other number of matches besides one.
+JsObject findRenderedDOMComponentWithClass(JsObject root, String className) {
+  return _TestUtils.callMethod(
+      'findRenderedDOMComponentWithClass', [root, className]);
+}
+
+/// Like scryRenderedDOMComponentsWithTag() but expects there to be one result,
+/// and returns that one result, or throws exception if there is any other
+/// number of matches besides one.
+JsObject findRenderedDOMComponentWithTag(JsObject element, String tag) {
+  return _TestUtils.callMethod(
+  'findRenderedDOMComponentWithTag', [element, tag]);
+}
+
+/// Same as scryRenderedComponentsWithType() but expects there to be one result
+/// and returns that one result, or throws exception if there is any other
+/// number of matches besides one.
+JsObject findRenderedComponentWithType(JsObject root, JsObject componentType) {
+  return _TestUtils.callMethod(
+      'findRenderedComponentWithType', [root, componentType]);
+}
+
+/// Returns true if element is a composite component.
+/// (created with React.createClass()).
+bool isCompositeComponent(JsObject element) {
+  return _TestUtils.callMethod('isCompositeComponent', [element]);
+}
+
+/// Returns true if instance is a composite component.
+/// (created with React.createClass()) whose type is of a React componentClass.
+bool isCompositeComponentWithType(JsObject element, JsFunction componentClass) {
+  return _TestUtils.callMethod(
+  'isCompositeComponentWithType', [element, componentClass]);
+}
+
+/// Returns true if instance is a DOM component (such as a <div> or <span>).
+bool isDOMComponent(Element element) {
+  return _TestUtils.callMethod('isDOMComponent', [element]);
+}
+
+/// Returns true if element is any ReactElement.
+bool isElement(element) {
+  return _TestUtils.callMethod('isElement', [element]);
+}
+
+/// Returns true if element is a ReactElement whose type is of a
+/// React componentClass.
+bool isElementOfType(JsObject element, JsFunction componentClass) {
+  return _TestUtils.callMethod('isElementOfType', [element, componentClass]);
+}
+
+/// Returns true if instance is a plain text component.
+bool isTextComponent(JsObject element) {
+  return _TestUtils.callMethod('isTextComponent', [element]);
+}
```

Comment:
I currently have no tests for ``isTextComponent()``. I couldn't figure out how it worked and it's not even listed on the <a href='http://facebook.github.io/react/docs/test-utils.html' target='_blank'>React Utils page</a>.